src/pages-and-resources/discussions/app-list/data/api.js

```diff
@@ -14,7 +14,13 @@ function normalizeApps(data) {
       id,
     })),
     appConfig: data.plugin_configuration,
-    activeAppId: data.providers.active,
+    // TODO: Defaulting to "legacy" should come from the backend.  For now this makes sense as we
+    // don't create a DiscussionsConfiguration object in the backend for legacy discussions, which
+    // is configured by default on all courses.  The endpoint doesn't know this and returns a null
+    // for `active`, which we can then interpret as "the user hasn't configured something else" and
+    // choose to fall back to legacy.  In the long run, we should be able to trust `active` to give
+    // us 'legacy' if the legacy experience is being used.
+    activeAppId: data.providers.active || 'legacy',
```

Comment:
@stvstnfrd take note!  Think this defaulting is fine for now as it says, but is something we'll ultimately want to be able to pull out.